src/settings-ui/Microsoft.PowerToys.Settings.UI/Strings/en-us/Resources.resw

```diff
@@ -956,4 +956,10 @@
   <data name="Run_NotAccessibleWarning.Text" xml:space="preserve">
     <value>Please define an activation phrase or allow this plugin for the global results to use it</value>
   </data>
+  <data name="Run_AllPluginsDisabled.Text" xml:space="preserve">
+    <value>PowerToys Run is not usable. Enable at least one plugin to use it.</value>
+  </data>
+  <data name="Run_NotAllowedActionKeyword.Text" xml:space="preserve">
+    <value>The action keyword is not allowed</value>

```

Comment:
I'd say "This action keyword is not allowed". Or ideally, show the keyword that was entered. E.g. "\\ is not allowed."

Reply:
Does it make sense to show the keyword. I mean the user can see the entered keyword two settings under this warning.
Regarding "this" and "the" I thought something similar. But I think it makes no difference.

But we should use the same wording in settings header and warning: "keyword" or "phrase"?

But should we note and hint on the fact, that the keyword works but can lead to problems on other plugins. I think this is an important point to know. So the we can write something like "This action phrase is not allowed and can brake other plugins."

Reply:
Good point

Reply:
@htcfreek
> This action phrase is not allowed and can brake other plugins.

It's a warning and it does not break other plugins. It overrides behavior. For example, I do not use `~` and instead, I want it to be an activation phrase, I should be allowed to do so. 

I think the correct message would be: "This activation phrase overrides the behavior of other plugins"